packages/wmr/src/lib/normalize-options.js

```diff
@@ -97,14 +96,81 @@ export async function normalizeOptions(options, mode) {
 
 	Object.defineProperty(options, '_config', { value: custom });
 	if (custom) {
-		if (custom.default) await custom.default(options);
-		if (custom[mode]) await custom[mode](options);
+		const fn = custom.default || custom[mode];
+		if (fn) {
+			const res = await fn(options);
+			if (res) {
+				options.plugins.push(res);
+			}
+		}
+	}
```

Comment:
We need to support running both `.default()` and `.[mode]()`:

```suggestion
		if (custom.default) {
			const res = await custom.default(options);
			if (res) options = mergeConfig(options, res);
		}
		if (custom[mode]) {
			const res = await custom[mode](options);
			if (res) options = mergeConfig(options, res);
		}
	}
```